### DISCUSSION TITLE
Training not generating metrics.val.npz for centered instance model

### DISCUSSION THREAD

**Discussion by clee758** [talmolab/sleap#1457] [General]
Hi, I trained a model, and the centroid model file has a metrics.val.npz, but the centered instance model keeps not generating one, and I need it to evaluate the metrics of the model. Am I doing something wrong and what can I do to fix this?

**Comment by roomrys** [comment]
Hi @clee758,

After the model is trained, are you getting predictions (with "Predict On" set to something)? One reason the centered_instance model may not be outputting a metrics file is because the model fails to predict on your images.

In the training pipeline, for the centered_instance model settings, are you checking the "Use Trained" checkbox? Another reason would be that the metrics would not be generated is if an already trained centered_instance model is being used for inference only.

Thanks,
Liezl

**Comment by clee758** [comment]
Hi, I tried both solutions but I keep having the same problem. Is there any other possibility?

**Reply by roomrys** [reply]
Hi @clee758,

Are able to upload a training package [here](https://urldefense.com/v3/__https://forms.gle/C5RGxse75MFEAUdn8__;!!GX6Nv3_Pjr8b-17qtCok029Ok438DqXQ!2tcmhPoVZuZXIr9bb5Jzcow13ul9w53GYiNBI8Dk_4-clDwK0ivSIdAQcSyZ13vb5mLx0G-95Y9m$) (export the training package from the training GUI) for me to troubleshoot?

Thanks,
Liezl
